cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -822,98 +829,108 @@ func (p *BicepProvider) Destroy(
 		return nil, fmt.Errorf("mapping resources to resource groups: %w", err)
 	}
 
+	// If no resources found, we still need to void the deployment state.
+	// This can happen when resources have been manually deleted before running azd down.
+	// Voiding the state ensures that subsequent azd provision commands work correctly
+	// by creating a new empty deployment that becomes the last successful deployment.
 	if len(groupedResources) == 0 {
-		return nil, fmt.Errorf("%w, '%s'", infra.ErrDeploymentResourcesNotFound, deploymentToDelete.Name())
-	}
+		p.console.StopSpinner(ctx, "", input.StepDone)
+		// Call deployment.Delete to void the state even though there are no resources to delete
```

Comment:
I'm wondering what does the printout look like if one were to run `azd down` repeatedly, on a environment that is already deleted.

Reply:
On the second `azd down`, it prompts for a location 🤔

<img width="1193" height="520" alt="image" src="https://github.com/user-attachments/assets/a8fbad6e-7162-44c5-9b25-12f58e17d0d2" />

After confirming a location, subsequent `azd down`s succeed normally:

<img width="1115" height="635" alt="image" src="https://github.com/user-attachments/assets/140700b3-ad03-41cd-b005-6cefb38de05a" />

Reply:
We can fix that prompt in a follow up PR